woocommerce-connect-client.php

```diff
@@ -747,9 +748,15 @@ public function hide_wc_connect_package_meta_data( $hidden_keys ) {
 		}
 
 		function hide_wc_connect_order_meta_data( $protected, $meta_key, $meta_type ) {
-			if ( 'wc_connect_labels' === $meta_key ) {
-				$protected = true;
+			switch ( $meta_key ) {
+				case 'wc_connect_labels':
+					$protected = true;
+					break;
+				case 'wc_connect_label_history':
+					$protected = true;
+					break;
```

Comment:
You can stack `switch` `case`s that have the same resulting block:
```javascript
switch ( $meta_key ) {
	case 'wc_connect_labels':
	case 'wc_connect_label_history':
		$protected = true;
		break;
}
```

Basically, if you remove the `break;`, the execution path just "falls through" to the next case.